readme.md

```diff
@@ -121,6 +121,7 @@ GitHub Enterprise is also supported. More info in the options.
 - [Quickly visit a repository's default branch and latest version tag.](https://user-images.githubusercontent.com/1402241/38107328-ccb3fb46-33bb-11e8-9654-23a6410943cc.png)
 - [Open multiple issues at once in your repos.](https://user-images.githubusercontent.com/1402241/38084752-4820b0d8-3378-11e8-868c-a1582b16f915.gif)
 - [Swap branches in the branch compare view.](https://user-images.githubusercontent.com/857700/42854438-821096f2-8a01-11e8-8752-76f7563b5e18.png)
+- [Exclude PR/issue filters from their list](https://user-images.githubusercontent.com/1402241/48470535-493cfb00-e824-11e8-863a-964f52b62553.png) with <kbd>alt</kbd> <kbd>click</kbd>.
```

Comment:
```suggestion
- [Exclude PR/issue filters from their list](https://user-images.githubusercontent.com/1402241/48470535-493cfb00-e824-11e8-863a-964f52b62553.png) with <kbd>alt/option</kbd>-<kbd>click</kbd>.
```

Reply:
Newer keyboards only say `option`. `option` is the name on macOS. I think we should respect that.

Reply:
I blame Tim Cook

Reply:
Wow this is annoying.

<img width="284" alt="screenshot 2018-11-29 at 15 32 28" src="https://user-images.githubusercontent.com/1402241/49206104-fe0e1500-f3eb-11e8-91ac-572c057d2d83.png">

I assigned myself but still...

<img width="638" alt="screenshot 2018-11-29 at 15 32 10" src="https://user-images.githubusercontent.com/1402241/49206110-ff3f4200-f3eb-11e8-8c3f-f78e5306de1f.png">

Reply:
Is that because it is a fork?
I think that would be a bug then?